src/modules/involvementApi.js

```diff
@@ -0,0 +1,41 @@
+// Involvement API methods access
+import { postData, getData } from './apiAccess.js';
+
+const INVOLVEMENT_API_URL = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi';
+
+const apiId = 'mJ38T1cF1k53CeEkLunA';
+const getLikes = async () => {
+  let answer = [];
+  const data = await getData(INVOLVEMENT_API_URL, `apps/${apiId}/likes`);
+  answer = data;
+
+  return answer;
```

Comment:
- [ ] **Optional:** The variable name `answer` doesn't really describe the content it holds. Kindly pick more descriptive variable names in the coming projects.